src/types/Review.ts

```diff
@@ -19,7 +20,7 @@ export type Review = {
   isPublished: boolean
   isPending: boolean
   ratings: Rating[]
-  reviewInput?: string
+  comment: Comment | null
```

Comment:
`comment?: Comment` should work here, unless you tried that and had problems?

Reply:
Along the way I had had problems; but at some point those disappeared since reverting back to optional is showing now problems.